codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientGenerator.kt

```diff
@@ -221,7 +221,7 @@ class FluentClientGenerator(
                 )
             }
         }
-        writer.withModule(RustModule.public("fluent_builders")) {
+        writer.withInlineModule(RustModule.public("fluent_builders")) {
```

Comment:
this isn't a change from this code—I just renamed to make it clearer. Inline module generates a `mod foo { ... }` block inside of another module instead of making a new file. I'm open to naming ideas